src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -26,8 +29,13 @@ public FindCommand parse(String args) throws ParseException {
         }
 
         String[] nameKeywords = trimmedArgs.split("\\s+");
+        List<String> fields = Arrays.asList(nameKeywords);
 
-        return new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
+        if (fields.get(0).substring(0, 1).equals("#")) {
```

Comment:
Are we going to enforce id to have '#' prefix? Because there isn't any constraint implemented yet